tools/ssb-tools/bin/gen-ssb-data.sh

```diff
@@ -126,7 +126,7 @@ date
 cd -
 
 # move data to $SSB_DATA_DIR
-mv "${SSB_DBGEN_DIR}"/*.tbl* "${SSB_DATA_DIR}/"
+mv "${SSB_DBGEN_DIR}"*.tbl* "${SSB_DATA_DIR}/"
```

Comment:
`"${SSB_DBGEN_DIR}"*.tbl*` looks weird. I think the original one is better. Maybe we should remove the `/` from the assignment of `SSB_DBGEN_DIR` in [#L32](https://github.com/apache/doris/blob/f8f4cba64f8e75910b0a447c1813a2203e5c99b4/tools/ssb-tools/bin/gen-ssb-data.sh#L32).

Reply:
ok, good idea, Thanks for your suggestion.